src/test/java/com/ably/kafka/connect/integration/ChannelSinkTaskTest.java

```diff
@@ -0,0 +1,295 @@
+ * do some assertions of our interest.
+ * System under test class is implicitly provided by connector configuration settings.
+ * See settings.put(CONNECTOR_CLASS_CONFIG, SINK_CONNECTOR_CLASS_NAME);
+ */
+
+public class ChannelSinkTaskTest {
+    private static final String CONNECTOR_NAME = "ably-test-connector";
+    private static final String SINK_CONNECTOR_CLASS_NAME = ChannelSinkConnector.class.getSimpleName();
+    private static final int NUM_WORKERS = 1;
+    private static final int NUM_TASKS = 1;
+    public static final long TIMEOUT = 5000L;
+    private static final String TOPICS = "topic1,topic2,topic3";
+    private static final String DEFAULT_TOPIC = TOPICS.split(",")[0];
+
+    private EmbeddedConnectCluster connectCluster;
+    private AblyHelpers.AppSpec appSpec;
+    private AblyRealtime ablyClient;
+
+    @BeforeEach
+    public void setup() throws Exception {
+        // create a test Ably app
+        appSpec = AblyHelpers.createTestApp();
+
+        // setup Connect cluster with defaults
+        connectCluster = new EmbeddedConnectCluster.Builder().build();
+
+        // start Connect cluster
+        connectCluster.start();
+        connectCluster.assertions().assertAtLeastNumWorkersAreUp(NUM_WORKERS, "Initial group of workers did not start in time.");
+
+        // connect Ably client
+        ablyClient = AblyHelpers.realtimeClient(appSpec);
+    }
+
+    @AfterEach
+    public void close() {
+        connectCluster.stop();
+        ablyClient.close();
+        AblyHelpers.deleteTestApp(appSpec);
+    }
+
+    // verify task is running
+    @Test
+    public void testMessagePublish_correctConfigAtLeastATaskIsRunning() throws Exception {
+        final String channelName = "test-channel";
+
+        connectCluster.kafka().createTopic(DEFAULT_TOPIC);
+        Map<String, String> settings = createSettings(channelName, null, null, null);
+        connectCluster.configureConnector(CONNECTOR_NAME, settings);
+        connectCluster.assertions().assertConnectorAndAtLeastNumTasksAreRunning(CONNECTOR_NAME, NUM_TASKS, "Connector tasks did not start in time.");
+
+        // delete connector
+        connectCluster.deleteConnector(CONNECTOR_NAME);
+    }
+
+    //channel name tests
+    @Test
+    public void testMessagePublish_channelExistsWithStaticChannelName() {
+        final String channelName = "test-channel";
+
+        connectCluster.kafka().createTopic(DEFAULT_TOPIC);
+        Map<String, String> settings = createSettings(channelName, null, null, null);
+        connectCluster.configureConnector(CONNECTOR_NAME, settings);
+
+        // subscribe to the Ably channel
+        Channel channel = ablyClient.channels.get(channelName);
+        AblyHelpers.MessageWaiter messageWaiter = new AblyHelpers.MessageWaiter(channel);
+
+        // produce a message on the Kafka topic
+        connectCluster.kafka().produce(DEFAULT_TOPIC, "foo", "bar");
+
+        // wait 5s for the message to arrive on the Ably channel
```

Comment:
This comment (and other instances of it) is not helpful for a few reasons... it:

1. repeats what the code already clearly tells us - it's superfluous - also, if `TIMEOUT` is change in the future then all instances of this comment will be incorrect
2. suggests to the reader that it "waits for 5 seconds", when that's technically wrong - it does wait but returns once the requisite number of messages have been received, therefore the time element is a _timeout_ 

I would remove these comments.

Reply:
This is also a good point 👍 I removed comments here https://github.com/ably/kafka-connect-ably/pull/49/commits/7d8905a51dd79bcd8e70820cfa05d020c7a029d1